Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidator.kt

```diff
@@ -39,13 +42,23 @@ class DccValidator @Inject constructor(
         return DccValidation(
             userInput = userInput,
             validatedAt = timeStamper.nowUTC,
+            signatureCheckPassed = signatureCheckPassed,
+            certificateExpiredAt = certificate.header.expiresAt.toLocalDateTime(DateTimeZone.UTC),
```

Comment:
Shouldn't this be converted using user tz as it is shown to the user? Or is there a requirement to use utc?

Reply:
Make sense